src/pls/data/nerd_icons.yml

```
@@ -5,31 +5,31 @@ nerd_icons:
   pls:        "\uf444"  # nf-oct-primitive_dot
 
   apple:      "\ue711"  # nf-dev-apple
-  audio:      "\ufc1d"  # nf-mdi-speaker_wireless
-  broom:      "\uf5e1"  # nf-mdi-broom
+  audio:      "\uf1c7"  # nf-fa-file_audio_o
+  broom:      "\ue615"  # nf-seti-config
```

There's no other broom, but stars convey the idea of "sparkling clean".
```suggestion
  broom:      "\ue370"  # nf-weather-stars
```